fix include statement for Boost header that was moved and is no longer available from 'tr1' in Boost 1.65.x and newer
author: Kenneth Hoste (HPC-UGent)
--- cufflinks-2.2.1/src/biascorrection.h.orig	2014-03-24 22:54:47.000000000 +0100
+++ cufflinks-2.2.1/src/biascorrection.h	2017-12-05 16:51:16.602586745 +0100
@@ -15,7 +15,7 @@
 #include <vector>
 #include <list>
 #include <string>
-#include <boost/tr1/unordered_map.hpp>
+#include <boost/unordered_map.hpp>
 #include <boost/thread.hpp>
 #include "common.h"
 
